question29/reset-email.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="ja">
+    <head>
+        <meta charset="UTF-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+        <link href="../css/style.css" rel="stylesheet" />
+        <link href="../css/common.css" rel="stylesheet" />
+        <link href="../css/form.css" rel="stylesheet" />
+        <script>
+            if (!localStorage.getItem("token")) window.location.href = "./login.html";
+          </script>
+        <title>TerraceTechフロントエンドエンジニア養成所</title>
+    </head>
+    <body>
+        <div id="js-wrapper" class="wrapper">
+            <h1 class="form_title">e-mail address Reset</h1>
+            <form method="post" id="form" novalidate>
+                <div>
+                    <ul>
+                        <li class="form_list">
+                            <label for="mail">新しいメールアドレス</label>
+                            <input type="email" id="mail" name="email" class="js-email" required>
+                            <span class="error" data-error="error"></span>
```

Comment:
IMO
Since reset-email.html is the page after login, we think it is easier for users to know where they are if it has the same header as the content page